release-notes/24.05/index.html

```diff
@@ -0,0 +1,150 @@
+            </li>
+            <li>
+            To strengthen library compartmentalization, we have completed an adjustment to the ABI for functions whose arguments do not fit in registers.  These arguments are now reachable via a bounded capability passed in register C9 and newly compiled callees access them via C9 allowing callers to place argument values anywhere they desire.  This is compatible with code compiled for 23.11, which used a transitional ABI which set C9 in the caller while the callee continued to access variables directly on the stack.  Code compiled for releases prior to 23.11 is no longer supported.
+            </li>
+            <li>
+            We have swiched the default kernel configuration to <code>GENERIC-MORELLO-PURECAP</code>, a pure-capability kernel.  This means that the entire base OS stack is running with spatial memory safety.  Performance of the pure-capability kernel is less well studied than the hybrid kernel so care should be taken when reporting results.
+            </li>
+            <li>
+            User level heap temporal safety performance has been improved by adding in-kernel worker threads.  This change allows userspace threads to invoke the kernel revoker, proceed with work, and then come back later to complete revocation.
+            </li>
+            <li>
+            Our CHERI-aware port of the bhyve hypervisor has been synced with AArch64 support merged to upstream FreeBSD.  The GDB stub has been enhanced to support reading capabilities in guest memory.
+            </li>
+            <li>
+            Kernels may now be compiled with the pure-capability benchmark ABI in addition to the existing hybrid and pure-capability ABIs. This ABI uses a modified form of code generation which improves performance on the Arm Morello board, required due to limitations with bounds prediction in the current Morello prototype. The performance of the benchmark ABI is more predictive of potential future CHERI microarchitectural performance than the Morello prototype running software compiled for CheriABI, making it preferable for benchmarking. However, this comes at the cost of reduced security, and so software compiled for the benchmark ABI should not be used for security evaluation. For details on how to use the benchmark ABI, and benchmark Morello in general, readers should see the Getting Started Guide.
```

Comment:
"may now" is a bit odd given we build and ship them to users

Reply:
I think the "may now" isn't related to the fact that we ship them but that users themselves can now do that type of compilation.  Unless I miss my guess.